test/functional/services/data_grid.ts

```diff
@@ -96,6 +98,42 @@ export function DataGridProvider({ getService }: FtrProviderContext) {
       await testSubjects.click(`dataGridHeaderCell-${columnName}`);
       await find.clickByButtonText('Remove column');
     }
+
+    async getDataGridTableTimestamp(): Promise<string[]> {
```

Comment:
So the primary difference between this one and getLastColumn() is the CSS class selector (`euiDataGridRowCell--date` vs. `euiDataGridRowCell--lastColumn`)?

Given this structure, it makes sense to refactor and abstract away the common logic into a single method. What do you think?

Something like 
```
async getDataGridTableSpecificColumnValues(columnSelector: string): Promise<string[]> {
    const table = await find.byCssSelector('.euiDataGrid');
    const $ = await table.parseDomContent();

    const columnValues: string[] = [];
    $.findTestSubjects('dataGridRowCell')
      .toArray()
      .forEach((cell) => {
        const cCell = $(cell);
        if (cCell.hasClass(`euiDataGridRowCell--${columnSelector}`)) {
          // The column structure may be very nested to get the actual text
          columnValues.push(cCell.children().children().children().children().text());
        }
      });

    return columnValues;
}
```

`columnSelector` could be `date` or `lastColumn`.

Reply:
+1 although id keep it simpler like `getDataGridTableColumn`